content/lab-wiki/code-of-conduct.md

```diff
@@ -0,0 +1,40 @@
++++ 
+title = "Code of Conduct" 
+category = "Lab Guide" 
```

Comment:
Thanks @bleds22e, This looks good to me but I am wondering if this first part was intended to be part of the document. let me know and we shall merge this